sympy/physics/mechanics/body.py

```diff
@@ -156,9 +156,19 @@ def z(self):
         """The basis Vector for the Body, in the z direction. """
         return self.frame.z
 
+    @property
+    def inertia(self):
```

Comment:
A simple docstring is needed for these properties. Should the be called central_inertia?

Reply:
No, `inertia` is overall defined as being a `tuple` `(Dyadic, Point)` (see `RigidBody`). However personally I would just most of the time set the central inertia of a body. The argument for `Body` also implies this preference, as it is `central_inertia` and not `inertia` like for `RigidBody`. This is also the reason why I've added a `central_inertia.setter` to `RigidBody`.